src/main/java/bio/terra/cli/service/SamService.java

```diff
@@ -425,20 +309,11 @@ private static boolean isNotFound(Exception ex) {
    */
   public void removeUserFromResource(
       String resourceType, String resourceId, String resourcePolicyName, String userEmail) {
-    ResourcesApi resourcesApi = new ResourcesApi(apiClient);
-    try {
-      HttpUtils.callWithRetries(
-          () -> {
-            resourcesApi.removeUserFromPolicy(
-                resourceType, resourceId, resourcePolicyName, userEmail);
-            return null;
-          },
-          SamService::isRetryable);
-    } catch (ApiException | InterruptedException ex) {
-      throw new SystemException("Error removing user from SAM resource.", ex);
-    } finally {
-      closeConnectionPool();
```

Comment:
Why don't you need to do this anymore?

Reply:
Same question, it's not clear to me that this happens now (but maybe that's OK?)

Reply:
Because now all methods in `SamService` and `WorkspaceManagerService` are now wrapped by the `handleClientExceptions` method, which includes this logic. My goal here was to stop duplicating this logic in each method and instead define it in a single place.

Reply:
This does still need to happen, otherwise the CLI hangs after a direct SAM call. I don't understand why this is required, but left a TODO in the code to revisit later.